assets/sass/v1/_idp-discovery.scss

```diff
@@ -0,0 +1,7 @@
+.idp-discovery {
+  .force-idp-spinner {
+    height: 38px;
+    margin-top: 40px;
+    background: url('../img/ui/indicators/loader@1x.gif') no-repeat center;
```

Comment:
duplicated definition with `v2/_device-challenge.scss`. we probably make a share component.